packages/samples/test/output/documentation/@typespec/openapi3/openapi.yaml

```diff
@@ -73,7 +73,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/Resp'
         '404':
-          description: The server cannot find the requested resource.
+          description: Not found
```

Comment:
See if this a correct change. This is due to comparing the response body with the response type and now it is different as some properites have been filtered out.